### PR TITLE
Added 2019 arXiv preprint publications.

### PR DESCRIPTION
**Pull request by dklong-csu**
As far as I can tell, the deletions are due to my text editor having slight formatting differences. It looks like tabs were used previously (possibly by me) and they were converted to spaces.

### CODE REVIEW

File: publications-2019.bib [hunk 1]
```diff
@@ -1057,4 +1057,273 @@ @PhdThesis{MB_PhD_Thesis
...
+    title={Simulating the scattering of low-frequency Gravitational Waves by compact objects using the finite element method},
+    author={Jian-hua He},
+    year={2019},
+    eprint={1912.00325},
+    archivePrefix={arXiv},
+    primaryClass={gr-qc}
+}
+
+
+@misc{mulita2019quasioptimal,
```
Comment (kronbichler) [inline publications-2019.bib:1170]: the publications starting from here and up to line 1267 appear to be duplicates.

**Comment by bangerth** [comment]
@dklong-csu Any news?

File: publications-2019.bib (hunk 1 above)
Comment (dklong-csu) [reply publications-2019.bib]: You're right, thanks for pointing those out.

File: publications-2019.bib [hunk 2]
```diff
@@ -1057,4 +1057,273 @@ @PhdThesis{MB_PhD_Thesis
...
+    year={2019},
+    eprint={1905.12996},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+
+@misc{badia2019generic,
+    title={A generic finite element framework on parallel tree-based adaptive meshes},
+    author={Santiago Badia and Alberto F. Martín and Eric Neiva and Francesc Verdugo},
```
[the remark this answers is not in the document]
Comment (dklong-csu) [reply publications-2019.bib]: Good catch. Updated in the most recent commit.

**Review by bangerth — APPROVED** [review]

**Pull request merged**